src/components/Extensibility/hooks/useDebugger.scss

```diff
@@ -0,0 +1,46 @@
+.debugonata {
+  width: auto;
+  max-height: 50vh !important;
+
+  h1 {
+    font-weight: bold;
+    border-bottom: 1px solid var(--sapTextColor);
+    padding: 0.5rem;
+  }
+
+  h2 {
+    margin-top: 0.5rem;
+    font-weight: bold;
+  }
+
+  .query {
+    padding-left: 1rem;
+  }
+
+  .info {
+    dt {
+      font-weight: bold;
+    }
+  }
+
+  .vars,
+  .data-sources {
+    padding-left: 1rem;
+
+    dd {
```

Comment:
maybe good option will be to make background for YAML code as sap-neutral or something like this, because right now it's unreadable